frb_codegen/src/library/codegen/generator/api_dart/text_generator.rs

```diff
@@ -47,8 +47,11 @@ fn generate_end_api_text(
         .join("\n\n");
     let classes = item.classes.iter().map(|c| c.code.clone()).join("\n\n");
 
-    // TODO use relative path calculation
-    let path_frb_generated = "../".repeat(namespace.path().len() - 2) + "frb_generated.dart";
+    let path = namespace.path();
+    let path_frb_generated = match path.len() {
+        1 => "rust/frb_generated.dart".to_string(),
```

Comment:
nit: codecov seems unhappy about this. maybe we should add a test to reproduce this scenario.

If I understand correctly, just write a function in `/Volumes/MyExternal/ExternalRefCode/flutter_rust_bridge/frb_example/pure_dart/rust/src/api/misc_example.rs`, and let that function have a struct in `src/lib.rs` as argument.